design-library/source/_patterns/01-atoms/08-logo/logo.json

```diff
@@ -0,0 +1,5 @@
+{
+  "image_url": "/images/nerdstein_11.png",
+  "image_alt": "GSA Logo",
```

Comment:
GSA?

Reply:
Ah yes, I had chosen the GSA logo when I was trying to turn this into a USWDS library ;)